Automattic/jetpack
# AI Assistant: Add ambiguous words to proofread features list

Fixes #38291

## Proposed changes:

* Adds types and fixes some ts issues
* Adds the ambiguous words feature

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor with the Breve flag enabled
* Check that the Ambiguous words checkbox is on the Jetpack AI sidebar, below the complex words
* Check that the toggle value is persisted on reload
* Add some words from the list, like "appear", and check that they are highlighted on the editor in green

| Menu | Highlight |
|---|---|
| ![2024-07-10_20-27-19](https://github.com/Automattic/jetpack/assets/8486249/407c6ce4-80ef-4fe9-bff6-a14987bdff50) | ![2024-07-10_20-27-48](https://github.com/Automattic/jetpack/assets/8486249/b513a401-bdb5-444c-b246-1700cef16669) |
